src/solvers/casadi.py

```diff
@@ -22,7 +22,7 @@ def solve_with_casadi(
     pf=20.76,
     pa=44.75,
     xi=0.01,
-    zeta=1.66e-4 * 1e9,  # zeta := 1.66e-4*norm_fac
+    zeta=1.66e-4 * 1e11,  # zeta := 1.66e-4*norm_fac
```

Comment:
same here.. why 1e11? not necessarily wrong, but add a comment explaining in case we forget in the future.